students/18th/18기 이대희/js/js:login.js

```diff
@@ -0,0 +1,25 @@
+const idinput = document.querySelector('.id');
+const pwinput = document.querySelector('.pw');
+const btn = document.querySelector('.btn');
+const rightbox1 = document.querySelector('.rightbox1');
+
+const changebtn = () =>{
+    let idinputval = idinput.value;
+    let pwinputval = pwinput.value;
+    if(!idinputval || !pwinputval){
+        btn.style.backgroundColor='rgb(192,223,253)';
+        console.log(btn.value);
+    }
+    else{
+        btn.style.backgroundColor='blue';
+    }
+}
```

Comment:
- 단일 if else문은 '삼항연산자'를 활용해 가독성 좋게 작성하실 수 있습니다.
- 더불어 조건문의 위치를 변경하여 중복되는 코드를 줄여볼 수 있겠네요.
```suggestion
   const activeCondition = !idInputVal || !pwInputVal
   btn.style.backgroundColor = activeCondition ? 'rgb(192,223,253)' : 'blue';
```

Reply:
수정했습니다.